docs/topics/testing/tools.txt

```diff
@@ -206,9 +211,18 @@ Use the ``django.test.Client`` class to make requests.
 
             name=fred&passwd=secret
 
-        If you provide ``content_type`` (e.g. :mimetype:`text/xml` for an XML
-        payload), the contents of ``data`` will be sent as-is in the POST
-        request, using ``content_type`` in the HTTP ``Content-Type`` header.
+        If you provide ``content_type`` as :mimetype:`application/json` the
+        POST data will be presented as a binary string like so::
+
+            {"name": "fred", "passwd": "secret"}
```

Comment:
I don't understand what this is demonstrating... this doesn't look like a binary string.

Reply:
Was split between representing it as an actual binary string, or what the binary string represents. I'll switch it to look like a binary string.